### PR TITLE
refactor(ui): move to screen based layout

### PR DESCRIPTION
**Pull request by xla**
*work in progress*

### CODE REVIEW

File: ui/DesignSystem/Component/Transaction/Center.svelte [hunk 1]
```diff
@@ -41,15 +44,13 @@
...
-    display: none;
-  }
 </style>
 
 <div class="center" class:negative data-cy="transaction-center">
-  <div class="list-wrapper" class:hidden>
-    <List on:select {transactions} />
-  </div>
+  {#if show}
+    <div class="list-wrapper" in:slide={listIn} out:slide={listOut}>
```
Comment (sarahscott) [inline ui/DesignSystem/Component/Transaction/Center.svelte:51]: these svelte animations are v nice

File: ui/App.svelte [hunk 2]
```diff
@@ -76,5 +79,8 @@
 <NotificationFaucet style="margin-top: calc(var(--topbar-height) + 11px)" />
 <Theme />
 <Remote {store} let:data={session} context="session">
-  <Router {routes} />
+  <TransactionCenter />
```
Comment (sarahscott) [inline ui/App.svelte:82]: I cleared my session and now this gets stuck on the `Sidebar` trying to render a `null` `identity`; might need to revisit how the layout renders before the `identity` exists.

File: ui/Screen/Settings.svelte [hunk 3]
```diff
@@ -44,117 +44,112 @@
...
-          </Text>
-          <Text>
-            This unlinks your local repositories from Upstream. The local data
-            will remain on your computer.
-          </Text>
-        </div>
-        <div class="action">
-          <Button variant="destructive">Remove</Button>
-        </div>
+  <section>
```
Comment (sarahscott) [inline ui/Screen/Settings.svelte:99]: getting real semantic html in here 👁️ very nice

File: ui/DesignSystem/Component/Sidebar.svelte [hunk 4]
```diff
@@ -15,16 +15,16 @@
 </script>
 
 <style>
```
Comment (sarahscott) [inline ui/DesignSystem/Component/Sidebar.svelte:17]: Not a huge fan of having 2 `Sidebar.svelte`'s in the project; maybe the layout one could be named something else? Like `MainLayout` or `HomeLayout` or `DefaultLayout`?

**Comment by xla** [comment]
Super-seeded by #582

**Pull request closed without merging**